### PR TITLE
Feed refactor

### PR DESCRIPTION
**Pull request by tbezman**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -339,7 +339,10 @@ type ViewerGallery {
 type Viewer {
     user: GalleryUser @goField(forceResolver: true)
     viewerGalleries: [ViewerGallery] @goField(forceResolver: true)
+
+    feed(before: String, after: String, first: Int, last: Int): FeedConnection
```
Comment (jarrel-b) [inline schema.graphql:343]: Not sure if this actually matters for the frontend, but we'll want `@goField(forceResolver: true)` for this as well

Comment (tbezman) [reply schema.graphql:343]: nice. makes sense. just a mock to test out my changes on front end. feel free to make whatever changes necessary on backend. I told @kaitoo1 already but I want to make sure we don't make any breaking changes so the website doesn't break for browsers loading the "old" client. 

TL;DR Three commits.
1. Make backend change adding new field to the viewer type
2. Make frontend change to use new field
3. Once 2 is deployed, we can remove the old field.

Comment (jarrel-b) [reply schema.graphql:343]: ah makes sense!

**Comment by jarrel-b** [comment]
Changes on the backend are now on dev to play with

File: schema.graphql (hunk 1 above)
Comment (Robinnnnn) [reply schema.graphql:343]: great call @jarrel-b, will apply!
